source/operations/server-side-encryption.rst

```diff
@@ -83,11 +88,73 @@ supports the following encryption strategies:
       SSE-C does not support bucket-default encryption settings and requires
       clients perform all key management operations.
 
-.. toctree::
-   :titlesonly:
-   :hidden:
+Configuring a KMS for MinIO
+---------------------------
 
-   /operations/server-side-encryption/configure-minio-kes-hashicorp
-   /operations/server-side-encryption/configure-minio-kes-aws
-   /operations/server-side-encryption/configure-minio-kes-gcp
-   /operations/server-side-encryption/configure-minio-kes-azure
\ No newline at end of file
+.. cond:: linux
+
+   This procedure provides guidance for deploying MinIO configured to use KES and enable :ref:`Server Side Encryption <minio-sse-data-encryption>`.
```

Comment:
We're saying "this procedure" but there are no steps below. We should at least have the MinIO steps right?